README.md

```diff
@@ -1,2 +1,200 @@
-# Startup-battlefield
+<a name="readme-top"></a>
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="logo-orange.png" alt="logo" width="160"  height="auto" />
+  <br/>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+  - [🚀 Loom Video](#loom-video)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Portfolio] <a name="about-project"></a>
+
+> # Startup-battlefield
 This is the official page of StartUp-Battlefield, a hackathon to build technologies for a sustainable growth to foster transparency and long-termism in financial and economic activity.
+
+ <img src="image.png" alt="logo" width="auto" />
+
+**[Portfolio setup and mobile first]** 
+
+## 🛠 Built With <a name="built-with">html, scss, bootstrap, javascript</a>
+
+### Tech Stack <a name="tech-stack">html, scss, bootstrap, javascript</a>
+
+> html, scss, bootstrap, javascript
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Turing Behance design to a project
+
+- **[Nav bar]**
+- **[About me section]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<<<<<<< HEAD
+<<<<<<< desktop-version-completed
+> Add a link to your deployed project.
+
+- [Live Demo Link]https://samtush.github.io/Startup-battlefield/
+<<<<<<< desktop-version-completed
+=======
+=======
+- [Live Demo Link](https://samtush.github.io/Startup-battlefield/)
+>>>>>>> main
+>>>>>>> main
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LOOM VIDEO -->
+
+## 🚀 Loom video <a name="loom-video"></a>
+
+- [Live Demo Link](https://www.loom.com/share/4a704710d1964b7390cc2a2003575079)
```

Comment:
- [ ] @SamTush great work in your presentation , however you forgot to include a section showing an interesting peace of code or code you proud of, kindly add it.